### ISSUE TITLE
Getting a "Can't resolve 'react-native'" error message

### ISSUE THREAD

**Issue by johot** [mobxjs/mobx-react-lite#8]
Great work getting this up an running @FredyC! However I am currently getting the following error message when creating a new project using create-react-app 2.1 (with `--typescript` not sure if it works without TypeScript):

```
Failed to compile.

./node_modules/mobx-react-lite/dist/index.module.js
Module not found: Can't resolve 'react-native' in 'C:\repos\testproj\node_modules\mobx-react-lite\dist'
```

Not sure what the correct fix for this is. I can't see the same type of solution in the old `mobx-react`. If I remove this from the dist it starts working again:

```
function configureNative() {
    const { unstable_batchedUpdates } = require("react-native");
    configure({ reactionScheduler: unstable_batchedUpdates });
}
```

**Comment by FredyC** [comment]
That's interesting. The `configureNative` is called withing try/catch block, but I suppose that Webpack sees the `require('react-native')` anyway and complains. There is a [similar code in mobx-react](https://github.com/mobxjs/mobx-react/blob/master/src/index.js#L4) but strangely enough, it's not present in bundled files. There is some magic happening I seem to have omitted :)

**Comment by FredyC** [comment]
Ok, the fact is even if I remove that code, the tests are passing, so why the hell not :) Will release a new version soon.

@mweststrate Can you give out hint, if the use `configure({ reactionScheduler: unstable_batchedUpdates  })` is actually useful for something in the scope of this package?

**Comment by johot** [comment]
Great @FredyC ! The bug also happened in non TypeScript fyi 😄

**Comment by FredyC** [comment]
Fixed in 2890b06d01436444486bd8618c41eb3be86f2d81 and v0.2.0 published, hope it works :)
